LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Authentication/DataBrokerProtectionSubscriptionManaging.swift

```diff
@@ -39,6 +39,7 @@ public final class DataBrokerProtectionSubscriptionManager: DataBrokerProtection
         // We use a staging token for privacy pro supplied through a github secret/action
         // for PIR end to end tests. This is also stored in bitwarden if you want to run
         // the tests locally 
+        return "anyStringAsATest"
```

Comment:
Temporary until I get them working again now that staging should be working again